src/redux/reducers/componentConfig/initialState.js

```diff
@@ -99,6 +99,41 @@ const heatmapInitialConfig = {
   truncatedValues: false,
 };
 
+// PLOTS & TABLES - Marker heatmap
+const markerHeatmapInitialConfig = {
```

Comment:
most of this config is the same as the config of `heatmapInitialConfig`, I would recommend that you modify the code here to copy whatever is in `heatmapInitialConfig` and then just build on top of it. so have something like:

```
const markerHeatmapInitialConfig = {
...heatmapInitialConfig,
groupedTracks: ['louvain'],
and so on
```